Tests/Unit/DependencyInjection/ONGRCurrencyExchangeExtensionTest.php

```diff
@@ -31,40 +31,17 @@ public function getTestLoadCurrencyData()
         $container1->setDefinition('ongr_currency_exchange.twig.price_extension', new Definition());
         $container1->setDefinition('stash.memcache', new Definition());
         $container1->setDefinition('my_service', new Definition());
```

Comment:
Why container1? are there more? In any case, I think that naming scheme {something}{1..n} should be avoided.